gpu-operator/google-gke.rst

```diff
@@ -67,11 +67,115 @@ Prerequisites
+
+#. Optional: Verify that you can connect to the cluster:
+
+   .. code-block:: console
+
+      $ kubectl get nodes -o wide
+
+#. Create the namespace for the NVIDIA GPU Operator:
+
+   .. code-block:: console
+
+      $ kubectl create ns gpu-operator
+
+#. Create a file, such as ``gpu-operator-quota.yaml``, with contents like the following example:
+
+   .. literalinclude:: ./manifests/input/google-gke-gpu-operator-quota.yaml
+      :language: yaml
+
+#. Apply the resource quota:
+
+   .. code-block:: console
+
+      $ kubectl apply -n gpu-operator -f gpu-operator-quota.yaml
+
+#. Optional: View the resource quota:
+
+   .. code-block:: console
+
+      $ kubectl get -n gpu-operator resourcequota
+
+   *Example Output*
+
+   .. code-block:: output
+
+      NAME                  AGE     REQUEST
+      gpu-operator-quota    38s     pods: 0/100
+
+#. Install the Google driver installer daemon set.
+
+   For Container-Optimized OS:
 
-Perform the following steps to create a GKE cluster with the ``gcloud`` CLI.
+   .. code-block:: console
+
+      $ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
+
+   For Ubuntu, the manifest to apply depends on GPU model and node version.
+   Refer to the **Ubuntu** tab at
+   `Manually install NVIDIA GPU drivers <https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#installing_drivers>`__
+   in the GKE documentation.
+
+#. Install the Operator using Helm:
+
+   .. code-block:: console
+
+      $ helm install --wait --generate-name \
+          -n gpu-operator \
+          nvidia/gpu-operator \
+          --set hostPaths.driverInstallDir=/home/kubernetes/bin/nvidia \
+          --set toolkit.installDir=/home/kubernetes/bin/nvidia \
+          --set cdi.enabled=true \
+          --set cdi.default=true \
+          --set driver.enabled=false
+
+   Set the NVIDIA Container Toolkit and driver installation path to ``/home/kubernetes/bin/nvidia``.
+   On GKE node images, the ``/home`` directory is writable and is a stateful location for storing the NVIDIA runtime binaries.
+
+   To configure MIG with NVIDIA MIG Manager, specify the following additional Helm command arguments:
+
+   .. code-block:: console
+
+      --set migManager.env[0].name=WITH_REBOOT \
+      --set-string migManager.env[0].value=true
+
+   Using NVIDIA MIG Manager to configure devices is an alternative to the default GKE approach and is incompatible with the default GKE approach.
+   Refer to `Running Multi-Instance GPUs <https://cloud.google.com/kubernetes-engine/docs/how-to/gpus-multi>`__
+   in the GKE documentation for information about the default approach.
+
+***************************
+Using NVIDIA Driver Manager
+***************************
+
+Perform the following steps to create a GKE cluster with the ``gcloud`` CLI and use the Operator and NVIDIA Driver Manager to manage the GPU driver.
```

Comment:
As a reader, it is not entirely clear to me that this section, `Using NVIDIA Driver Manager`, is only applicable for Ubuntu node pools. It may be worth adding a sentence here stating that this approach is only supported on Ubuntu.